contentcuration/contentcuration/migrations/0098_auto_20190213_2236.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2019-02-13 22:36
+from __future__ import unicode_literals
+
+import contentcuration.models
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contentcuration', '0097_task'),
+    ]
+
+    operations = [
+        migrations.AddField(
```

Comment:
Hmm is there a way we can squash migrations?

Reply:
Yes, I think so, I can go ahead and do that.

Reply:
Do you know a good way to test migrations? When I used `squashmigrations`, it created a squashed migration and left the other two migrations in the tree. It said to leave them for a while and manually remove them when you know it's safe to do so. I'd want to make sure that manually changing the migration wouldn't break anything. Another option is just to never squash during development, but periodically squash old migrations, maybe ones 6 months old or greater?

Reply:
@kollivier we can remove the other migrations, since it should always take the squashed migration path now. And no one else has used your code anyway!